# 予想

1. try文を実行。
2. 例外をスローしたので、catch節の例外処理を実行
3. breakを実行するが、catch節で例外が処理された後にfinally節が必ず実行されるので、continueが実行される。
4. continueが実行され、次のループに進む

上記の順番で進み、for文はbreakすることなく最後まで実行するので、i = 5となり5が出力される

# 実行結果

> 5
